dockers/docker-lldp/lldpmgrd

```diff
@@ -219,6 +242,18 @@ def main():
 
     lldpmgr.run()
 
+def resume_lldpd(self):
+    cmd = "lldpcli resume"
+    proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Please exctract "run_cmd" code in a function. And use the function in both places.

Reply:
Fixed, please check